### ISSUE TITLE
Publish x64 version auth dll with amd64 suffix to match os.arch

### ISSUE THREAD

**Issue by mwisnicki** [microsoft/mssql-jdbc#1905]
Can you change arch suffix of x64 version to amd64 such that it matches ${os.arch}?

This will make usage a bit easier:

```
<dependency>
    <groupId>com.microsoft.sqlserver</groupId>
    <artifactId>mssql-jdbc_auth</artifactId>
    <version>11.2.0.${os.arch}</version>
    <type>dll</type>
</dependency>
```

Otherwise I have to have an extra maven profile to map os.arch to suffix.

**Comment by Jeffery-Wasty** [comment]
Hi @mwisnicki,

We can take a look at this in our upcoming triage, but generally small-scale enhancements like this are lower priority. If we end up addressing this, we may look at making the auth.dll names configurable, such as is suggested in #1850. Outright changing the name is probably not happening.

**Comment by mwisnicki** [comment]
Hey, thanks for reply. I don't think making names configurable at runtime changes anything with regard to this request as the artifact published in maven still needs to be named uniquely (unless you change how you distribute dlls).

**Comment by Jeffery-Wasty** [comment]
That was also discussed as part of the issue. Regardless, we'll take a look at all of our naming at a later time.

**Comment by lilgreenbird** [comment]
hi @mwisnicki 

ultimately we would like to get rid of the DLLs all together however we are currently blocked by issue AzureAD/microsoft-authentication-library-for-java#299. Once that's available we will be working towards moving all the DLL functionalities into the Java code so we can get rid of all the DLLs.

**Comment by TarasTielkes** [comment]
Will a Windows arm64 version of `mssql-jdbc_auth` be provided in a future release?

**Comment by Jeffery-Wasty** [comment]
@lilgreenbird's comment above still stands, our next goal regarding these auth DLLs is their removal entirely, however we're still blocked by the PR mentioned above.

**Comment by TarasTielkes** [comment]
Out of curiosity, how does `microsoft-authentication-library-for-java` eliminate the need for JNI to call `crypt`/`advapi`?
